### PR TITLE
[enterprise-4.9] Add 4.9 ShiftStack release notes

### PR DESCRIPTION
**Pull request by maxwelldb**
4.9 RN additions for ShiftStack. Will need dev input on this--a lot of 4.9 in ShiftStack was no-doc.

Preview link: https://deploy-preview-36531--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes

### CODE REVIEW

**Comment by maxwelldb** [comment]
Here's a place to stash 4.9 ShiftStack RNs. Feel free to use the suggestion tool or PRs to make changes. 

Adding folks who own/reported 4.9-targeted epics for review: @mandre @iamemilio @mdbooth @pierreprinetti @adduarte

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -147,6 +152,14 @@ The CNI VRF plug-in was previously introduced as a Technology Preview feature in
 
 For more information, see xref:../networking/multiple_networks/assigning-a-secondary-network-to-a-vrf.adoc#cnf-assigning-a-secondary-network-to-a-vrf[Assigning a secondary network to a VRF].
 
+[id="ocp-4-9-networking-openshift-on-openstack-cloud-provider-options"]
+==== Load balancer configuration as a cloud provider option for clusters on {rh-openstack}
+
+For clusters that run on {rh-openstack}, you can now configure Octavia for load balancing as a cloud provider option.
```
Comment (mandre) [inline release_notes/ocp-4-9-release-notes.adoc:158]: Perhaps this is a bit confusing, as it was already possible to configure the load balancer previously. In 4.9, we've changed the default to enable the use of octavia when applicable, and have QE coverage to support multiple scenario.
There's also desire to support the Octavia load balancer in previous releases (provided manual configuration).

**Review by mandre — COMMENTED** [review]
Thanks @maxwelldb. I believe this covers everything. Adding @dulek to get Kuryr's perspective too.
/cc @dulek

File: release_notes/ocp-4-9-release-notes.adoc (hunk 1 above)
Comment (maxwelldb) [reply release_notes/ocp-4-9-release-notes.adoc:158]:
```suggestion
For clusters that run on {rh-openstack}, Octavia is now available as a default load balancer option. 
```

@mandre ?

**Comment by pierreprinetti** [comment]
:+1:

File: release_notes/ocp-4-9-release-notes.adoc [hunk 2]
```diff
@@ -147,6 +152,14 @@ The CNI VRF plug-in was previously introduced as a Technology Preview feature in
 
 For more information, see xref:../networking/multiple_networks/assigning-a-secondary-network-to-a-vrf.adoc#cnf-assigning-a-secondary-network-to-a-vrf[Assigning a secondary network to a VRF].
 
+[id="ocp-4-9-networking-openshift-on-openstack-cloud-provider-options"]
+==== Load balancer configuration as a cloud provider option for clusters on {rh-openstack}
+
+For clusters that run on {rh-openstack}, Octavia is now available as a default load balancer option. 
+
+// TODO: Link!
+// For more information, see xref:../
```
Comment (dulek) [inline release_notes/ocp-4-9-release-notes.adoc:161]: I added a known issue about this feature into the tracker PR recently. :/ Here's the BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2004542.

Comment (maxwelldb) [reply release_notes/ocp-4-9-release-notes.adoc:161]: TY. :\

**Comment by maxwelldb** [comment]
Created https://github.com/openshift/openshift-docs/issues/37071 as a reminder to add a link when available.

**Pull request merged**